### DISCUSSION TITLE
upgrade to 6.0.x

### DISCUSSION THREAD

**Discussion by rsedlmeier** [firefly-iii/firefly-iii#7390] [Firefly III]
Tried to install the data-importer, however it says:

Your Firefly III version 5.7.18 is below the minimum required version 6.0.5

Havnt sufficient git experiences, can anyone give a hint how to upgrade my firefly version w/o overwriting the current?

Many thanks
R.

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/installation/upgrade/
